pkg/mount/raw_mount.go

```diff
@@ -0,0 +1,158 @@
+// +build linux
+
+package mount
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/docker/docker/pkg/mount"
+	"github.com/libopenstorage/openstorage/pkg/keylock"
+	"github.com/sirupsen/logrus"
+)
+
+// rawMounter loads mounts for raw volumes that are bind mounted in the mount table
+type rawMounter struct {
+	Mounter
+}
+
+// NewRawBindMounter returns a new rawBindMounter
+func NewRawBindMounter(
+	rootSubstrings []string,
+	mountImpl MountImpl,
+	allowedDirs []string,
+	trashLocation string,
+) (*rawMounter, error) {
+	b := &rawMounter{
+		Mounter: Mounter{
+			mountImpl:     mountImpl,
+			mounts:        make(DeviceMap),
+			paths:         make(PathMap),
+			allowedDirs:   allowedDirs,
+			kl:            keylock.New(),
+			trashLocation: trashLocation,
+		},
+	}
+	if err := b.Load(rootSubstrings); err != nil {
+		return nil, err
+	}
+	return b, nil
+}
+
+func (b *rawMounter) Reload(rootSubstring string) error {
+	newRBM, err := NewRawBindMounter(
+		[]string{rootSubstring},
+		b.mountImpl,
+		b.allowedDirs,
+		b.trashLocation,
+	)
+	if err != nil {
+		return err
+	}
+	return b.reload(rootSubstring, newRBM.mounts[rootSubstring])
+}
+
+// this mount filtering implementation is done based on logic implemented in findmnt + libmount
+func (b *rawMounter) Load(rawVolumeDevicesPaths []string) error {
```

Comment:
These require some unit tests

Reply:
Will add some tests.

Reply:
Added tests in `mount_test.go` similar to other mounters @lpabon.

Reply:
Having trouble with these tests on Travis. Working on this now.

Reply:
Got this passing, had to switch to a test directory that was mounted in the container `/mnt/...`